tests/end_to_end/http_server/hybrid_canister/benchmarks.md

```diff
@@ -0,0 +1,86 @@
+# Benchmarks for server
+
+## Current benchmarks Azle version: 0.25.0
+
+| Id  | Method Name         | Instructions  | Cycles        | USD           | USD/Million Calls | Change                     |
+| --- | ------------------- | ------------- | ------------- | ------------- | ----------------- | -------------------------- |
+| 0   | init                | 8_135_030_419 | 6_454_602_167 | $0.0085824909 | $8582.4909        | <font color="red">0</font> |
```

Comment:
Snake case for USD only before the decimal

Reply:
done